flask_parameter_validation/parameter_types/parameter.py

```diff
@@ -4,6 +4,8 @@
 """
 import re
 from datetime import date, datetime, time
+from enum import Enum, StrEnum, IntEnum
```

Comment:
Risky StrEnum as per comment

Reply:
What do you think about performing the import, catching `ImportError` if it is raised, on which we fall back on checking for `issubclass(allowed_types[0], Enum) and issubclass(allowed_types[0], str)` to replace the `StrEnum` import, and `issubclass(allowed_types[0], Enum) and issubclass(allowed_types[0], int)` to replace the `IntEnum` import?

Reply:
Sounds like a great solution, especially if also compatible with StrEnum anyway :D

Reply:
Implemented ✅